selfdrive/modeld/runners/thneedmodel.pxd

```diff
@@ -0,0 +1,9 @@
+# distutils: language = c++
+#cython: language_level=3
+
+from libcpp.string cimport string
+from cereal.visionipc.visionipc cimport cl_context
+
```

Comment:
a bunch of the diff here is just wrapper. can we reduce this at all? at least collapse into a single pxd/pyx pair?

Reply:
Not easily :/ thneedmodel has to be its own pxd file since it's only compiled under certain conditions, onnxmodel and snpemodel could technically be combined but since we're going to get rid of onnxmodel as soon as navmodeld and dmonitoringmodeld are rewritten (which I'm going to do right after this), I don't think it really makes sense to combine them.